### PR TITLE
Return Genset.genes as sorted list instead of set

### PR DESCRIPTION
**Pull request by JureZmrzlikar**
@mzganec - please test:
1. If it satisfies your needs better
2. For potential bugs :-)

### CODE REVIEW

File: src/resdk/resources/geneset.py [hunk 1]
```diff
@@ -20,15 +21,15 @@ def __init__(self, resolwe, genes=None, source=None, species=None, **model_data)
         """Initialize attributes."""
         self.logger = logging.getLogger(__name__)
 
-        self._genes = None
-        # Make sure genes are stored in a set object
-        if genes is not None:
-            self._genes = set(genes)
+        super().__init__(resolwe, **model_data)
```
Comment (AGregorc) [inline src/resdk/resources/geneset.py:24]: That should probably be here from the beginning 😅

**Review by AGregorc — APPROVED** [review]
I played with Genesets a bit and didn't find any bugs 👍

**Comment by JureZmrzlikar** [comment]
@mzganec 
Ping.

**Review by mzganec — APPROVED** [review]
After multiple conversations with our data science team members a decision has been made to return genesets as native python lists. @JureZmrzlikar Thanks for preparing the code as desired.

**Pull request merged**